src/style.css

```diff
@@ -0,0 +1,44 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--2 .stars__star:nth-child(n+1):nth-child(-n+2) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--3 .stars__star:nth-child(n+1):nth-child(-n+3) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(n+1):nth-child(-n+4) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
+
+.stars--5 .stars__star:nth-child(n+1):nth-child(-n+5) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+}
```

Comment:
check out point 2 in the [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

Reply:
fixed the bug, thanks